MAUI/DataForm/DataForm-settings.md

```diff
@@ -0,0 +1,899 @@
+
+## Change the Background for editor layout
+
+You can change the background color for the label and editor by using `GenereateDataFormItem` event when `AutoGenerate` is true.
+
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.Background = Brush.Azure; 
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+GenerateDataFormItem event when `AutoGenerate` is true.
+
+For `AutoGenerate` false you can use property directly by using the DataFormItem - add reference link for explicit creaate data editor
+
+## Change the order for editor
+
+You can change the row order of editor using `RowOrder` property of `DataFormViewItem` by using GenerateDataFormItem event when `AutoGenerate` is true.
+
+For `AutoGenerate` false you can use property directly by using the DataFormItem - add refrence link for explict create data editor
+
+You can change the order of item's within a row using `ItemsOrderInRow` property, which is applicable only when `ColumnCount` is set to more than one. You can place an item within a row from 0 to `ColumnCount` - 1 position.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.ColumnCount = 3;
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.RowOrder = 1;
+                e.DataFormItem.ItemsOrderInRow = 2;  
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+GenerateDataFormItem event when `AutoGenerate` is true.
+
+For `AutoGenerate` false you can use property directly by using the DataFormItem - add reference link for explicit creaate data editor
+
+## Add Label icon for the editor
+
+You can set label icon instead of using label text by using GenereateDataFormItem event when `AutoGenerate` is true.
+
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "PhoneNumer")
+            {
+                e.DataFormItem.LeadingLabelIcon = new FontImageSource() 
+                { 
+                    Glyph = "K", 
+                    Color = Colors.Black, 
+                    FontFamily = "DataFormIcons", 
+                    Size=18 
+                };
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+GenerateDataFormItem event when `AutoGenerate` is true.
```

Comment:
GenerateDataFormItem event when `AutoGenerate` is true.

For `AutoGenerate` false you can use property directly by using the DataFormItem - add reference link for explicit creaate data editor

remove